src/ucp/tag/rndv.c

```diff
@@ -79,15 +103,16 @@ static size_t ucp_tag_rndv_rts_pack(void *dest, void *arg)
     rndv_rts_hdr->sreq.reqptr      = (uintptr_t)sreq;
     rndv_rts_hdr->sreq.sender_uuid = ep->worker->uuid;
     rndv_rts_hdr->size             = sreq->send.length;
-    if (UCP_DT_IS_CONTIG(sreq->send.datatype)) {
+    if (UCP_DT_IS_CONTIG(sreq->send.datatype) &&
+        (ep->worker->context->config.ext.rndv_mode == UCP_RNDV_MODE_GET_ZCOPY ||
+         ep->worker->context->config.ext.rndv_mode == UCP_RNDV_MODE_AUTO)) {
```

Comment:
Does current implementation assume `AUTO` == `GET_ZCOPY`? maybe better to handle this on initialization? I mean 
```
if (rndv_mode == UCP_RNDV_MODE_AUTO) {
    rndv_mode = UCP_RNDV_MODE_GET_ZCOPY;
}
```